akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -227,8 +261,12 @@ object ClusterSingletonManager {
       final case class OldestChanged(oldest: Option[UniqueAddress])
     }
 
+    final case class AcquireLeaseResult(holdingLease: Boolean)
+    final case class ReleaseLeaseResult(released: Boolean)
+    final case class LeaseLost(reason: Option[Throwable])
```

Comment:
`DeadLetterSuppression` on these since they are piped and actor may have been terminated